docs/backends/geoip.rst

```diff
@@ -165,6 +173,12 @@ Keys explained
 
   :services: Defines one or more services for querying.
              Each service name may have one or more placeholders.
+  :mapping_lookup_formats: Defines which format to interpolate when using the ``%mp`` placeholder. Each entry
+                          is looked up in the given order and stops at first match.
+                          This allows using a fine granularity, (e.g. per country), while limiting the number
+                          of records to create.
+                          You can use any placeholder within the given format (e.g. %cc).
```

Comment:
After a bit of thinking I think you should forbid the `%mp` placehold in lookup formats to avoid infinite recursion.

Reply:
Added check at loading time, and we now throw an exception of `%mp` is in the lookup formats.